stronk/models/workout_exercise.py

```diff
@@ -20,6 +33,37 @@ class WorkoutExercise(db.Model):
     workout_reps = db.Column(ARRAY(db.Integer), nullable=False)
     rest_time = db.Column(db.Integer, nullable=False)
 
+    @staticmethod
+    def create(workout_id, exercise_id, workout_weights, workout_reps, rest_time):
+        workoutExercise = WorkoutExercise(
+            workout_id=workout_id,
+            exercise_id=exercise_id,
+            workout_weights=workout_weights,
+            workout_reps=workout_reps,
+            rest_time=rest_time if rest_time else 0
+        )
+
+        if (len(workout_weights) != len(workout_reps)):
+            raise BadAttributes(MISMATCHED_LENGTHS_ERROR_MSG)
+
+        try:
+            db.session.add(workoutExercise)
+            db.session.commit()
+
+            return workoutExercise
+        except IntegrityError as err:
+            if isinstance(err.orig, UniqueViolation):
+                raise Conflict(
+                    "Exercise has already been added to this workout")
+            elif isinstance(err.orig, ForeignKeyViolation):
+                raise BadAttributes("User or exercise not found")
```

Comment:
Should it be Workout or Exercise not found since we need either a valid exercise or workout id?

Reply:
Yup mb - good catch - that's a typo!